nvidia.go

```diff
@@ -0,0 +1,116 @@
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"golang.org/x/sys/unix"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+const ldconfPath = "/etc/ld.so.conf"
+
+func setupNvidiaDriver(containerRootfs string) (err error) {
+	bins, libs, err := getNvidiaFiles()
+	if err != nil {
+		return
+	}
+
+	// mount driver binaries
+	for _, binFile := range bins {
+		if err = mount(binFile, containerRootfs+binFile, "bind", unix.MS_BIND|unix.MS_REC, "rbind"); err != nil {
```

Comment:
how about using `filepath.Join(containerRootfs, binFile)`?  
security concern: have you tried to modify the binary form the container? I'd prefer to copy the binary

Reply:
To fix.(filepath.Join)
As for copying or mounting, I  have tested to modify the binary form the container, it can't be modified because it's read only( as the vm is read only file system). And the binaries are also isolated by kata vm. Change in one container will not affect other containers. So I used mounting.